### PR TITLE
Harden github workflow against injection

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: src/locales/de/translation.json [hunk 1]
```diff
@@ -129,10 +129,6 @@
     "switchToMainnet": "Wechsel zu Mainnet."
   },
   "footer": {
-    "poweredBy": {
-      "oasismonitor": "Powered by Oasis Monitor API & Oasis gRPC",
-      "oasisscan": "Powered by Oasis Scan API & Oasis gRPC"
-    },
```
Comment (lubej) [inline src/locales/de/translation.json:135]: Not sure this change belongs to this PR.

**Review by lubej — APPROVED** [review]

File: src/locales/de/translation.json (hunk 1 above)
Comment (tjanez) [reply src/locales/de/translation.json:135]: Agreed. Was this an unintentional change?

File: .changelog/2033.trivial.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Harden github workflow against injection
```
Comment (tjanez) [inline .changelog/2033.trivial.md:1]: This could also deserve a mention in the internal section of the CHANGELOG

**Review by tjanez — APPROVED** [review]
Thanks for thinking about security!

**Pull request merged**